chi_edge/cli.py

```diff
@@ -178,24 +266,24 @@ def bootstrap(
     ),
 )
 def bake(
-    device_uuid: "str" = None,
+    device_uuid: "str",
     image: "str" = None,
 ):
     # Ensure we do not overwrite a `config.json` file on the user's system
     if os.path.isfile("config.json"):
-        print("'config.json' already exists!")
-        return
+        raise click.ClickException("'config.json' already exists!")
 
     # Check for device in doni
     doni = doni_client()
     hardware = doni.get(f"/v1/hardware/{device_uuid}/").json()
     balena_workers = [
-        worker for worker in hardware["workers"]
-        if worker["worker_type"] == "balena"
+        worker for worker in hardware["workers"] if worker["worker_type"] == "balena"
     ]
     if not balena_workers:
-        print(f"Expected to find 1 balena worker, found {len(balena_workers)}")
-        return
+        raise click.ClickException(
+            "Device is not enrolled with support for Balena, you may need to attempt "
```

Comment:
Will users know what Balena is when they see this error message?

Reply:
We mention it in the docs, I think they will have some minimal knowledge. Practically this shouldn't happen at all, just being specific so they can attach a more detailed error message.